result/yoda.js

```diff
@@ -0,0 +1,18 @@
+
+/*eslint no-multi-spaces: "error"*/
+
+var a =  1;
+var a =  1;
+var a =  1;
+
+if(foo   === "bar") {}
+
+a <<  b
+a <<  b
```

Comment:
<!-- Comment by [SideCI](https://sideci.com) -->

This comment should be deleted too